.github/workflows/ci.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           npm install
           npm run riffraff-artefact
   ScalaBuild:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
I can see this jumping to an ubuntu version without java 8? (I'm not sure how this interacts with the mozilla sbt image, so this may not be useful)